<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaTipoUsuarios</class>
 <widget class="QMainWindow" name="VistaTipoUsuarios"> <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width> <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POWERGYM - Seleccionar Tipo de Usuario</string> </property>
  <property name="styleSheet"> <string notr="true">
    QMainWindow#VistaTipoUsuarios { /* Usamos el objectName de la ventana principal */
        background-color: #f0f0f0; /* Color de fondo gris claro */
    }
    QWidget#centralwidget {
        background-color: #f8f8f8; /* Color de fondo del widget central, un poco más claro */
        padding: 20px; /* Padding general para el contenido */
    }
    
    QLabel { /* Estilo general para todos los QLabel */
        color: #333333;
        font-size: 14px;
        font-weight: 500;
    }

    QLabel#labelTituloPrincipal { /* Estilo específico para el nuevo título principal */
        color: #B71C1C; /* Rojo oscuro para el texto del título */
        font-size: 28px; /* Tamaño de fuente grande */
        font-weight: bold;
        padding-bottom: 20px;
    }

    QPushButton {
        background-color: #D32F2F; /* Rojo vibrante */
        color: white;
        border: none;
        border-radius: 20px; /* Bordes más redondeados */
        padding: 15px;
        font-size: 16px;
        font-weight: bold;
        min-height: 40px; /* Altura mínima para botones */
        margin-top: 15px; /* Margen superior para separar botones */
    }
    QPushButton:hover {
        background-color: #C62828; /* Rojo más oscuro al pasar el ratón */
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_main"> <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>

    <item>
     <widget class="QLabel" name="labelTituloPrincipal"> <property name="text">
       <string>Seleccionar Tipo de Usuario</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>

    <item>
     <spacer name="verticalSpacer_top_buttons"> <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item>
     <widget class="QPushButton" name="botonAtletas">
      <property name="text">
       <string>Ver Atletas</string>
      </property>
     </widget>
    </item>

    <item>
     <widget class="QPushButton" name="botonEntrenadores">
      <property name="text">
       <string>Ver Entrenadores</string>
      </property>
     </widget>
    </item>

    <item>
     <spacer name="verticalSpacer_middle_buttons"> <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

    <item>
     <widget class="QPushButton" name="botonVolver">
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </item>

    <item>
     <spacer name="verticalSpacer_bottom"> <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>